code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -761,10 +761,10 @@
 	user.visible_message("<span class='notice'>[user] opens \the [src] and starts pumping the handle.</span>", \
 						"<span class='notice'>You open \the [src] and start pumping the handle.</span>")
 	while(recharging)
-		if(!do_after(user, 10, src))
+		if(!do_after(user, 10, src, ignore_movement = TRUE))
 			break
 		playsound(get_turf(src),'sound/items/change_drill.ogg',25,1)
-		if(power_supply.give(60) < 60)
+		if(power_supply.give(90) < 90)
```

Comment:
This should probably be exported to a var. Still, I would rather do small, adjustments. Make the main phaser receive 75 charge with each pump. Make the holdout receive 100 charge with each pump. The key concern here is that I do not want the phasers to become better than military weapons.

Reply:
How exactly do I do that? There's no individual var under the holdout versus regular phasers.

Reply:
Same way you did for the crank time in the prior PR. Add a new /var/energy = 75 (probably not energy since that var may be used by something else) in the initial phaser. Replace 90 with energy, and then in the holdout phaser do energy = 100

Reply:
Updated with /var/phase_power = 75, and set holdout to 100